TODO.txt

```diff
@@ -0,0 +1,126 @@
++ cmake --build ./cmake-build-fmt-x86_64-Debug/install
+[1/12] Building CXX object test/CMakeFiles/test-main.dir/util.cc.o
+[2/12] Building CXX object test/CMakeFiles/test-main.dir/gtest-extra.cc.o
+[3/12] Building CXX object test/CMakeFiles/test-main.dir/test-main.cc.o
+[4/12] Building CXX object test/gtest/CMakeFiles/gtest.dir/gmock-gtest-all.cc.o
+[5/12] Building CXX object test/CMakeFiles/module-test.dir/test-main.cc.o
+[6/12] Linking CXX static library test/gtest/libgtest.a
+[7/12] Linking CXX static library test/libtest-main.a
+[8/12] Building CXX object test/CMakeFiles/test-module.dir/__/src/fmt.cc.o
+[9/12] Building CXX object CMakeFiles/fmt.dir/src/fmt.cc.o
+[10/12] Building CXX object test/CMakeFiles/module-test.dir/module-test.cc.o
+[11/12] Linking CXX static library libfmtd.a
+[12/12] Linking CXX executable bin/module-test
+FAILED: bin/module-test
+: && /usr/local/opt/llvm/bin/clang++-16 -g -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  test/CMakeFiles/test-module.dir/__/src/fmt.cc.o test/CMakeFiles/module-test.dir/module-test.cc.o test/CMakeFiles/module-test.dir/test-main.cc.o -o bin/module-test  test/libtest-main.a  test/gtest/libgtest.a && :
+Undefined symbols for architecture x86_64:
+  "fmt::v9::vsystem_error(int, fmt::v9::basic_string_view<char>, fmt::v9::basic_format_args<fmt::v9::basic_format_context<fmt::v9::appender, char>>)", referenced from:
+      std::__1::system_error fmt::v9::system_error<>(int, fmt::v9::basic_format_string<char>) in libtest-main.a(gtest-extra.cc.o)
+  "fmt::v9::file::dup(int)", referenced from:
+      output_redirect::output_redirect(__sFILE*) in libtest-main.a(gtest-extra.cc.o)
+  "fmt::v9::file::dup2(int)", referenced from:
+      output_redirect::output_redirect(__sFILE*) in libtest-main.a(gtest-extra.cc.o)
+      output_redirect::restore() in libtest-main.a(gtest-extra.cc.o)
+  "fmt::v9::file::pipe(fmt::v9::file&, fmt::v9::file&)", referenced from:
+      output_redirect::output_redirect(__sFILE*) in libtest-main.a(gtest-extra.cc.o)
+  "fmt::v9::file::read(void*, unsigned long)", referenced from:
+      output_redirect::restore_and_read() in libtest-main.a(gtest-extra.cc.o)
+      read(fmt::v9::file&, unsigned long) in libtest-main.a(gtest-extra.cc.o)
+  "fmt::v9::file::close()", referenced from:
+      fmt::v9::file::operator=(fmt::v9::file&&) in libtest-main.a(gtest-extra.cc.o)
+      output_redirect::restore() in libtest-main.a(gtest-extra.cc.o)
+      output_redirect::restore_and_read() in libtest-main.a(gtest-extra.cc.o)
+  "fmt::v9::file::~file()", referenced from:
+      output_redirect::output_redirect(__sFILE*) in libtest-main.a(gtest-extra.cc.o)
+      output_redirect::~output_redirect() in libtest-main.a(gtest-extra.cc.o)
+ld: symbol(s) not found for architecture x86_64
```

Comment:
Das sieht so aus, als ob `format.o` fehlt.
Der Test hat alle Symbole doppelt: einmal im Modul `fmt` and damit attached an das Modul mit anderem Mangling, und ein zweites Mal in der Test-Infrastruktur (`libtest-main` mit `gtest`und `fmt`) als traditioneller Build mit normalem Mangling.